docs/20_tutorial.md

```diff
@@ -1,64 +1,245 @@
+RLA process. 
+
+While the RLA Tool does provide for upload of the 
+hashed Ballot Manifest, the hashed CVR file and their hashes, 
+the RLA Tool does not provide a hashing utility.
+
+The RLA Tool does not provide any support
+for chain of custody logs. Nor does it provide any guidance for ballot 
+interpretation. 
+
+## For Secretary of State Users
+
+### Logging In
+
+Screenshot (SoS authentication) ---
+
+### Identifying County Users
+
+The Secretary of State will determine, for each individual
+County, which user will be able to log in to the County-facing
+part of the RLA Tool.
+
+Screenshot (SoS County User Identification)
+
+### Entering the Risk Limit
+
+To enter the Risk Limit for comparison audits:`
+Screenshot (SoS dash?) --- 
+
+### Entering the Random Seed
+
+To enter the random seed:
+Screenshot (SoS dash?) ---
+
+### Selecting Contests
+
+To select the contests that will drive the sample size and stop/go
+decisions for each round:
+Screenshot (SoS dash?) ---
+
+### Reading the Number of Ballots to Audit
+
+The RLA Tool calculates the number of ballots to audit in each round of the RLA. To read
+the number of ballots for each round:
+Screenshot (SoS dash?) ---
+
+### Declaring a Full Hand Count
+
+Screenshot ---
 
-***should the run book cover the mechanics of storing ballots and the
-other physical aspects of the RLA process which are not ‘in’ the RLA
-tool/system?
 
 Basic overview: Screenshots of the dashboard/tool, ordered in the sequence
 they will be encountered, with brief, clear instructions describing what
 is expected of the user(s) at each step, starting with the login screen
 and proceeding through the completion of the audit, or determination that
 a full hand count is required. First SOS, then County, then Public.
 
-Detailed instructions with associated screenshot about the process of
-Audit Board members signing in.
+## For County Administrators
+
+For the 2017 Colorado implementation, the full functionality of the RLA Tool requires exports from the 
+Dominion System ???. 
+***Colorado2017
+***which Dominion system?
+
+### Logging In
+
+Screenshot (County Admin Authentication) ---
+
+
+### <a name="comparison-audit-upload"></a>Comparison Audit Upload
+
+To prepare for upload the 
+ballot manifest and the CVR file must be first verified and hashed. 
+The RLA Tool does not provide a utility for the 
+verification and hashing. 
+***ballot manifest upload prep
 
-Detailed descripton of the SOS dashboard with associated screenshot with
-descriptions of all fields and/or information/updates.
+Once the ballot manifest and CVR files are verified and hashed, 
+they can be uploaded.
 
-Detailed description with associated screenshot of the County dashboard
-with instructions about the process of uploading ballot manifests and
-CVRs, and associated hash functions.
+Screenshot (page:county) ---
+
+Note that the RLA Tool does not support export of the
+tabulation results to the Election Night Reporting System. 
```

Comment:
Paragraph 1: you and I have different concepts of the audience and purpose of the Run Book. It's a limited, streamlined document, simply designed to guide the people who have to interact directly with the computer system. It's not going to be read cover to cover but used as a reference as people learn to use the computer system.

Paragraph 2: point taken.

Paragraph 3: Out of scope for the Run Book. See Paragraph 1.